docs/framework/guides/contributing/git-commit-message-convention.md

```diff
@@ -18,7 +18,7 @@ Every commit made *directly* to the `master` branch must follow the below conven
 Commit message template:
 
 ```
-Type: A short sentence about the commit. Closes #XXX.
+Type (package-name, other-package-name): A short sentence about the commit. Closes #XXX.
```

Comment:
We need this mentioned somewhere that we should trip the `@ckeditor/ckeditor5-` and `@ckeditor/ckeditor-` prefixes. And that the main package is referenced as `ckeditor5`.

Reply:
BTW, we weren't supposed to handle multiple packages in a single changelog item: https://github.com/ckeditor/ckeditor5/issues/6058#issuecomment-611482923